data/src/main/java/com/example/data/mappers/WeatherDataEntityMapper.kt

```diff
@@ -58,10 +62,17 @@ class WeatherDataEntityMapper(private val cityConverter: CityConverter) {
                 temp = it.temperature,
                 icon = it.icon,
                 date = parsingDate(it.date),
-                description = it.description
+                description = it.description,
+                temperatureType = getUnitType()
```

Comment:
There is a duplicating code, for all fields. To prevent it, the method fromEntityToWeatherData can be used here in map.

Reply:
Fixed